src/3rd_party/CMakeLists.txt

```diff
@@ -235,7 +235,8 @@ if (NOT ${Boost_FOUND})
   include(ExternalProject)
   ExternalProject_Add(
     Boost
-    URL https://dl.bintray.com/boostorg/release/1.66.0/source/boost_1_66_0.tar.gz
+    URL https://mirror.bazel.build/dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.gz
+    URL_HASH SHA256=da3411ea45622579d419bfda66f45cd0f8c32a181d84adfa936f5688388995cf
```

Comment:
This change doesn't seem relevant to the feature. Could you please explain why this change is required? If its not relevant to the feature it should be moved to a different PR.

Reply:
@ShobhitAd This change was required because the boost serves was unavailable for some time, so we have added the temporary commit where we added the link to the boost library mirror - 3db9681 
It should be removed before the merge.